internal/importer/importer.go

```diff
@@ -34,7 +34,7 @@ func New(filename string, setters ...Option) (Importer, error) {
 	case format.CSV:
 		return NewCSVImporter(filename, opts.Sheets, opts.Parser)
 	case format.XML:
-		return NewXMLImporter(filename, opts.Sheets, opts.Header)
+		return NewXMLImporter(filename, opts.Sheets)
```

Comment:
Maybe you should delete the related code at https://github.com/tableauio/tableau/blob/master/internal/importer/options.go#L29